### ISSUE TITLE
need to be updated

### ISSUE THREAD

**Issue by madakab** [ambozz/linkvertise-bypass#2]
I tried it today and it seems needs an update

**Comment by NSPC911** [comment]
Yes, this is what happened to me, it was working until the latest Firefox Update when it just decided to stop working. Please try and fix it. I use it a lot and having this break means I have to resort to other heavy scripts

**Comment by visible-sounds** [comment]
Please update

**Comment by askfriends** [comment]
+1
